src/test/unit/extension.test.ts

```diff
@@ -31,8 +31,7 @@ describe("extension", () => {
     it("should display simple loading message when branch name is yet to load", async () => {
```

Comment:
this test i an old unit test runnig with jest that will be refactored to mocha in unit tests ticket